### ISSUE TITLE
Create API leaves Web Service URL blank

### ISSUE THREAD

**Issue by leerodgers** [stephaneey/azure-apim-extension#21]
Creating a new API now seems to work propertly where it takes all of the settings but I would expect the base URL to use the base URL of the swagger.  For example:

if I used this URL for swagger: https://testing123.azurewebsites.net/swagger/ui/index

I would expect the baseUrl to be https://testing123.azurewebsites.net/

Instead the baseUrl is blank and that results in my API being set up to go nowhere.  Is this working by design?

**Comment by leerodgers** [comment]
It actually keeps wiping out the base URL, i have to disable the task.  I don't think it was doing this yesterday.

**Comment by leerodgers** [comment]
by Base URL i mean: Web service URL

**Comment by ohdihe** [comment]
Hi @leerodgers 
I am having the same issue when I use the **Create or Update API task**. However, it works as expected when I use the **Create or Update Versioned API task**. 

Something must have gone wrong somewhere. I am just assuming, based on the time stamp of the code commits and the time stamp of your issue ticket.

**Comment by stephaneey** [comment]
Hi @ohdihe and @leerodgers ,

Sorry guys! I had the authorization feature that disappeared probably due to a conflict in a pull request that I had overlooked. I integrated back but used the wrong verb (PUT instead of PATCH) to update the API with the authorization bits, that caused the service url to vanish...So, it's back to normal now. I just republished the extension on the marketplace and I'm going to push the commit now.

Best Regards
